av.h

```diff
@@ -236,7 +236,12 @@ to fit one element without extending:
 
 */
 
-#define newAV()	MUTABLE_AV(newSV_type(SVt_PVAV))
+/* sv_upgrade does AvREAL_only():
```

Comment:
What does this comment refer to?

Reply:
I just moved the existing comment from mathoms.  It was added by

> commit ac572bf4fde55cf563ae338dfe4e12b22ebc6d80
 Author: Nicholas Clark <nick@ccl4.org>
 Date:   Thu Dec 20 19:49:50 2007 +0000
       Perl_newAV() can become a mathom by making newAV() a wrapper around
     newSV_type() and tweaking Perl_sv_upgrade().
          p4raw-id: //depot/perl@32675

Reply:
I see. That comment stopped making sense after b9f83d2f7c, when the `sv_upgrade` calls were removed.

Reply:
I removed the entire comment